utils/data.cpp

```diff
@@ -0,0 +1,253 @@
+#include "data.hpp"
+#include <QDebug>
+
+namespace jbrush {
+
+// Constructor makes some simple assignments
+Data::Data(Canvas* theCanvas) : mainCanvas(theCanvas), currentDirectory(STARTING_DIRECTORY){}
+
+void Data::Load()
+{
+    // Bring up a panel for the user to select a directory
+    currentDirectory = QFileDialog::getOpenFileName(mainCanvas, "Please Choose a File", STARTING_DIRECTORY, "Text files (*.txt)");
```

Comment:
We should define our own file format, just for fun :D

Reply:
You mean instead of *.txt?  How do you DO that?!